client/Hooks/useLogger/useLogger.steps.tsx

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+import { And, Given, When, Then, Fusion, After } from 'jest-cucumber-fusion';
+import { renderHook } from '@testing-library/react-hooks';
+import { Server } from 'mock-socket';
+
+import { useLogger, LoggerType, MESSAGE_BUFFER_MAX_SIZE } from '.';
+
+let logger: LoggerType;
+let rerenderHook: (newProps?: unknown) => void;
+let mockWebSocketServer: Server;
+let onConnectionPromise: Promise<void>;
+let onMessagePromise: Promise<string>;
+let sentMessagesCount = 0;
+let verifiedClient = true;
+
+After(() => {
+  sentMessagesCount = 0;
+  mockWebSocketServer.close();
```

Comment:
May be undefined if the `Given` was never called

Reply:
Check added